### DISCUSSION TITLE
How to use pandoc_args lua-filter in quarto

### DISCUSSION THREAD

**Discussion by jiangshiguo** [quarto-dev/quarto-cli#4247] [Q&A] [answered]
I used to use pandoc_args lua-filter (see examples from [Section 5.1](https://bookdown.org/yihui/rmarkdown-cookbook/font-color.html#lua-color) of the RMarkdown Cookbook) to set text colors in bookdown books. How can I use it in pandoc?

Thank you!

**Comment by mcanouil** [comment, marked as answer]
This is a LUA filter, see <https://quarto.org/docs/extensions/filters.html> for more.

**Reply by jiangshiguo** [reply]
Thank you for pointing to this! I remember going through the Quarto documentation and obviously overlooked/skipped this section. I saved the `color-text.lua` file in the project folder and added the following to the `_quarto.yml` file and it works like a charm.

```
filters:
  - color-text.lua
```

So now the top section of my  `_quarto.yml` file  looks like this:

```
project:
  type: book

filters:
  - color-text.lua
```
